.github/workflows/publish.yml

```diff
@@ -4,9 +4,8 @@ name: Build macOS and Windows
 
 # Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
+  # Triggers the workflow on release tags
```

Comment:
It doesn't run this workflow in this repo for some reason... ICBINS has issues with this line.

All the other repos release without branch specifier
```
name: Publish
on:
  push:
    tags:
      - "*"
``` 

Maybe something to do with the Jenkins